### ISSUE TITLE
Discrepancy between size estimates/calculations

### ISSUE THREAD

**Issue by satra** [dandi/dandi-api#260]
when i use the stats api, i get:
```
{
"dandiset_count":65
"published_dandiset_count":0
"user_count":66
"size":16363640643492
}
```
when i sum all the content of the size of dandisets by going through, i get:
15414647730404

```python
import requests
url = "https://api.dandiarchive.org/api/dandisets/?page_size=100"
response = requests.request("GET", url)
dandisets = response.json()["results"]
ds = []
for val in dandisets:
    ds.append({"identifier": val["identifier"], "size": val["draft_version"]["size"], 
               "updated": val["draft_version"]["modified"]})
df = pd.DataFrame(ds)
df["size"].sum()
```

**Comment by yarikoptic** [comment]
which dandiset that would be?   should also match (in humanized form) to the stats on https://github.com/dandisets/000006/tree/57eb5db01fc0b08cd3ff3a67e80cc3fc5d6fd57f I guess

**Comment by satra** [comment]
> which dandiset that would be?

what do you mean @yarikoptic ?

**Comment by yarikoptic** [comment]
I thought it was stats on some particular dandiset, missed that it is for the entire archive.  Indeed must be counting at the level of blobs disregarding that some of those might belong to no longer linked to any dandiset assets.
FWIW, the ultimate code should be done at the level of DB query which would first get all unique blobs assigned to any versioned dandiset, and sum their sizes: so we do not limit ourselves to drafts, but also do not double count blobs reused across versions or (later) dandisets.  Difference from total size of blobs would also be informative on how much we could GC.

**Comment by satra** [comment]
i just checked blobs on AWS and it is 14.8TB, so i don't think total size is the sum of blobs.

**Comment by satra** [comment]
i believe this was addressed in #307, but yet to see effect on production.
